packages/web-app-webfinger/src/discovery/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface OwnCloudInstance {
+  rel: string
+  href: string
+  titles?: any
```

Comment:
Can we specify the type a bit more, or can it literally be anything?

Reply:
was just too lazy during development 😁  changed to `Record<string, string>`